package.json

```diff
@@ -12,18 +12,16 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@finos/perspective-webpack-plugin": "0.4.1",
-    "@grafana/data": "next",
-    "@grafana/toolkit": "next",
-    "@types/react": "^16.9.19",
+    "@grafana/toolkit": "canary",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.2",
-    "jest-canvas-mock": "^2.2.0"
+    "enzyme-adapter-react-16": "^1.15.2"
   },
   "dependencies": {
-    "@finos/perspective-viewer": "0.4.1",
-    "@finos/perspective-viewer-d3fc": "0.4.1",
-    "@finos/perspective-viewer-hypergrid": "0.4.1",
-    "@grafana/runtime": "next",
-    "@grafana/ui": "next"
+    "@finos/perspective-viewer": "0.4.3",
+    "@finos/perspective-viewer-d3fc": "0.4.3",
+    "@finos/perspective-viewer-hypergrid": "0.4.3",
+    "@grafana/data": "canary",
+    "@grafana/runtime": "canary",
+    "@grafana/ui": "canary"
```

Comment:
Lets move this back to `devDependencies` since they are not bundled with the final plugin (external in the webpack config)